org.opent2t.sample.lamp.superpopular/com.wink.lightbulb/js/thingTranslator.js

```diff
@@ -149,6 +112,43 @@ class Translator {
                 return response.power.value;
             });
     }
+
+    // Helper method to convert the device schema to the translator schema.
+    deviceSchemaToTranslatorSchema(deviceSchema) {
```

Comment:
>deviceSchemaToTranslatorSchema [](start = 4, length = 30)

add static to both of these functions so you don't need an instance to reference. We have an example of this in the OpenT2T library also (it's in ts but when you compiles shows you the js syntax)
[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static)